webtau-testapp/src/main/java/com/example/demo/springboot/app/SpringBootDemoApp.java

```diff
@@ -17,7 +18,9 @@
 @Import({springfox.documentation.spring.data.rest.configuration.SpringDataRestConfiguration.class})
 public class SpringBootDemoApp {
     public static void main(String[] args) {
-        SpringApplication.run(SpringBootDemoApp.class, args);
+        ConfigurableApplicationContext context = SpringApplication.run(SpringBootDemoApp.class, args);
+        String port = context.getEnvironment().getProperty("local.server.port");
+        System.out.println("Springboot test server running on port " + port);
```

Comment:
do we still need this line?

Reply:
That's debatable.  The Maven plugin for reserving the port logs the number so it's probably not needed.  I'll remove it.